CMakeLists.txt

```diff
@@ -212,6 +221,31 @@ if(UMF_FORMAT_CODE_STYLE)
             --i
             ${format_list}
         COMMENT "Format files using clang-format")
+
+    # Obtain files for cmake-format check
+    set(format_cmake_glob)
+    foreach(DIR IN ITEMS include src test benchmark)
```

Comment:
you're missing a few dirs (`cmake`, `examples` and also root dir..?)

Please, sort them A-Z